### ISSUE TITLE
Remote configuration failure

### ISSUE THREAD

**Issue by hasikp1** [thingsboard/thingsboard-gateway#326]
I am trying to push configuration from Thingsboard CE to gateway. I  am getting following.
```
Jun 25 12:04:01 raspberrypi python3[2041]: ""2020-06-25 12:04:01" - INFO - [tb_device_mqtt.py] - tb_device_mqtt - 194 - connection SUCCESS"
Jun 25 12:04:01 raspberrypi python3[2041]: ""2020-06-25 12:04:01" - ERROR - [tb_gateway_remote_configurator.py] - tb_gateway_remote_configurator - 79 - 'utf-8' codec can't decode byte 0x89 in position 0: invalid start byte"
Jun 25 12:04:01 raspberrypi python3[2041]: Traceback (most recent call last):
Jun 25 12:04:01 raspberrypi python3[2041]:   File "/usr/lib/python3/dist-packages/thingsboard_gateway/gateway/tb_gateway_remote_configurator.py", line 59, in process_configuration
Jun 25 12:04:01 raspberrypi python3[2041]:     self.__new_configuration = loads(decoded_configuration)
Jun 25 12:04:01 raspberrypi python3[2041]:   File "/usr/lib/python3/dist-packages/simplejson/__init__.py", line 516, in loads
Jun 25 12:04:01 raspberrypi python3[2041]:     return _default_decoder.decode(s)
Jun 25 12:04:01 raspberrypi python3[2041]:   File "/usr/lib/python3/dist-packages/simplejson/decoder.py", line 373, in decode
Jun 25 12:04:01 raspberrypi python3[2041]:     s = s.decode(self.encoding)
Jun 25 12:04:01 raspberrypi python3[2041]: UnicodeDecodeError: 'utf-8' codec can't decode byte 0x89 in position 0: invalid start byte
```
Extensions are not synced.

**Comment by zbeacon** [comment]
Hi @hasikp1 

Please tell more, how do you try to push the configuration?

**Comment by hasikp1** [comment]
I am using Thingsboard CE and I have created a device and set it as gateway. From the extensions in the gateway properties I have configured modbus RTU extension with required settings such as polling time, id, register addresses and saved it.

![extensions](https://user-images.githubusercontent.com/48294216/85670940-040acb80-b6df-11ea-9133-d63d8ec3b122.PNG)

**Comment by zbeacon** [comment]
It is old remote configuration, it will be deprecated in the future versions. The dashboard for remote gateway configuration is available in version 2.5 and above (If you install demo data).
